packages/vite-plugin-cloudflare/src/utils.ts

```diff
@@ -1,5 +1,27 @@
+import * as path from 'node:path';
 import { invariant } from './shared';
-import type { WorkerOptions } from 'miniflare';
+import type { SharedOptions, WorkerOptions } from 'miniflare';
+
+const DEFAULT_PERSIST_PATH = '.wrangler/state/v3';
+
+export function getPersistence(
+	persistTo: string | false | undefined,
+	root: string,
+): Partial<SharedOptions> {
+	if (persistTo === false) {
+		return {};
```

Comment:
sorry I don't fully remember Miniflare's default behavior, are you sure this does what we expect to? or should we clearly set the various persistance fields to `false`?

Reply:
This does do what we expect because `undefined` is equivalent to `false` here.

https://github.com/cloudflare/workers-sdk/tree/main/packages/miniflare#type-persistence

I guess you could argue that setting each value to `false` explicitly is more resilient if this behaviour ever changed. What do you think?

Reply:
Ah ok, thanks for checking that! all good then 😄 

> I guess you could argue that setting each value to false explicitly is more resilient if this behaviour ever changed. What do you think?

mh.... yeah I do think that that would be more resilient, but it would also be more verbose since you'd have to add all the various fields here 😕 (without an immediate benefit to back this up)

So if you do want to me more explicit that's fine by me, but I'm also ok if we want to have more concise code, I'm happy with whatever you prefer 🙂 (PS: even if miniflare's default behavior were to change and this would break it would be unlikely to create much damage and the fix would be pretty trivial)

Reply:
OK, sure. I've made the changes. As I've used the `PersistOptions` type without `Partial` we'd get a type error if Miniflare stopped accepting `undefined` for persist values. I think that's fine without the extra explicitness.